dockerfile/dockerfile_raspberrypi

```diff
@@ -6,33 +6,24 @@ RUN apt-get update
 RUN apt-get install -y \
     build-essential \
     cmake \
-    curl \
+    debootstrap \
     git \
-    libcurl4-openssl-dev \
-    libssl-dev \
-    make \
-    pkg-config \
-    uuid-dev \
-    valgrind
+    nodejs \
+    npm \
+    qemu-user-static
+
+ENV SYSROOT /root/rpi/chroot-raspbian-armhf
 
-RUN apt-get install -y nodejs \
-    npm    
 
 RUN git clone https://github.com/yungez/rpitools.git
```

Comment:
Yes, it will be in the coming commit, for building image and testing takes a while.